packages/verification/src/rest/v1/verifications/verifications-api.ts

```diff
@@ -254,6 +254,11 @@ export class VerificationsApi extends VerificationDomainApi {
       'Content-Type': 'application/json; charset=UTF-8',
       'Accept': 'application/json',
     };
+    if (data.startVerificationWithSmsRequestBody.smsOptions?.locale !== undefined) {
+      headers['Accept-Language'] = data.startVerificationWithSmsRequestBody.smsOptions?.locale;
+      // Remove the `locale` property from the body as it is used as a header parameter for the API call
+      delete data.startVerificationWithSmsRequestBody.smsOptions?.locale;
```

Comment:
If we are deleting body filled by SDK user we could break it implementation (i.e. same object reused within a loop)

Reply:
Arf, you're right, I should have used a copy